docs/articles/blog/2020-9-2-how-testcafe-can-help-you-to-deliver-faster.md

```diff
@@ -0,0 +1,87 @@
+
+TestCafe ships with five reporters – modules that can generate test execution reports for your CI/CD. JSON, JUnit, xUnit, and console output (with rich formatting) are built-in. You can also use reporters developed by the TestCafe user community for [NUnit](https://www.npmjs.com/package/testcafe-reporter-nunit), [TeamCity](https://www.npmjs.com/package/testcafe-reporter-teamcity), [Slack](https://www.npmjs.com/package/testcafe-reporter-slack), etc. Should you require these community plugins, you can:
+
+* Install the desired third-party reporter via npm: `npm install testcafe testcafe-reporter-nunit`;
+* Run TestCafe tests much like step 3: `testcafe chrome:headless ./tests -r nunit:report.xml`
+
+You can learn more and about TestCafe and CI system integration via the following links:
+
+* Jenkins
+* GitHub Actions
+* Travis
+* CircleCI
+* TeamCity
+* GitLab
+* Azure DevOps
+* BitBucket Pipelines
+* AppVeyor
+
+Note: For Jenkins, TestCafe can post videos and screenshot it captured during test execution. To enable this functionality, you will need to install [the Jenkins plugin](https://plugins.jenkins.io/testcafe/) and use a dedicated [Jenkins reporter](https://www.npmjs.com/package/testcafe-reporter-jenkins).
+
+## Docker Image
+
+TestCafe is easy to install and requires a very basic environment. Of course, you can wind it up even faster with Docker. To get started, download a pre-configured image from Docker Hub:
+
+```sh
+docker pull testcafe/testcafe
+```
+
+The image is based on ArcLinux and includes Node.js, TestCafe, Chrome, and Firefox. It is ready to execute tests as needed:
+
+```sh
+docker run -v //d/tests:/tests -it testcafe/testcafe firefox:headless /tests/**/*.js
+```
+
+You can also use this image on your computer to perform tests on a host machine or remote machines within your network. See the following help topic for additional Docker-related information: [Use TestCafe's Docker Image](https://devexpress.github.io/testcafe/documentation/guides/advanced-guides/use-testcafe-docker-image.html)
+
+## GitHub Actions
+
+The GitHub Actions CI service is ideal for projects developed on GitHub. To help you quickly integrate TestCafe into your Actions workflows, we created the [Run TestCafe](https://github.com/DevExpress/testcafe-action) action. This action installs TestCafe and runs tests on your behalf. To begin, you simply need to add TestCafe command line arguments:
+
+```yaml
+- uses: DevExpress/testcafe-action@latest
+  with:
+    args: "chrome tests"
+```
+
+## TestCafe: Adoption Made Easy
+
+Ready to see TestCafe's capabilities in action? Want to see how quickly you can incorporate test automation into your CI/CD workflow?
+
+[Get Started Today](https://devexpress.github.io/testcafe/documentation/getting-started/) or [Ask Us](https://devexpress.com/ask) for more information. We are here to help.
```

Comment:
Why we need to lead people to SC here? Maybe https://devexpress.github.io/testcafe/support/ is better.